prog.py

```diff
@@ -27,8 +27,13 @@ def read_from_file(file_name) -> tuple:
 
 
 def user_input():
-    phone_number = input("Please enter phone")
-    message = input('Please enter a new message (type quit to exit): ')
+
+    phone_number = int(input("Please enter phone: "))
```

Comment:
If the entered phone number is not a valid value for `int`, this statement will immediately raise a ValueError.
The point of input validation is to provide users with an easily identifiable cause of the error, and a chance to fix it, if possible. So this whole piece would rather look like:
```
phone_number = input("Please enter phone: ")
try:
    phone_number = int(phone_number)
except ValueError:
   raise ValueError(f"The entered phone number {phone_number} is not a number")
# this will only happen if there was no error on the previous step
message = input('Please enter a new message (type quit to exit): ')
```